.github/workflows/build.yml

```diff
@@ -25,14 +25,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # Node 14 matches the version of Node used by VS Code when this was
-      # written, but it should be updated when VS Code updates its Node version.
+      # updated to Node 16
```

Comment:
I'd prefer keeping the above comment, but just updating to say Node 16 so it's clear we're trying to sync with the VS Code Node version.